jobs/mozillians_magic.rb

```diff
@@ -42,19 +41,22 @@ def query_mozillians(params = {})
       http.request(request)
     end
 
-    def query_mozillians_group(group)
-      response = query_mozillians({ group: group })
-
-      if response.code.to_i == 200
-        res = JSON.parse(response.body)
-        count = res['count']
+    def iterate_mozillians_groups(groups)
+      remove_from_group('mozillians_nda')
```

Comment:
nit: You can change the `review_from_group` method to accept a list of groups and parse the groups in that method. With this change you can avoid calling the method per group/multiple times. feel free to ignore this.